### ISSUE TITLE
Unwrenched hydroponics trays can be stacked together on one tile and will work they same as if they were wrenched

### ISSUE THREAD

**Issue by despotate** [d3athrow/vgstation13#13723]
I mentioned this on someone else's issue but it needs its own report. You can plant, weed, water, farmbot, etc., any amount of trays stacked on one tile and they'll grow plants just fine, though I believe they won't process while inside a crate. Also, you can wrench down multiple trays on the same tile if you want to.

This exploit makes it easy to do hydroponics for me, just unwrench all the trays and vendors and move them near the fridge, mass a huge amount of plants without taking a single step. Also makes it easy to grow shit as a trader on the vox shuttle, just unwrench the trays and bring them onto the shuttle in a crate. Finally, one bee apiary can cover infinite amounts of hydroponics trays with this method.

**Comment by Exxion** [comment]
The day this is exploited in a round-destroying-yet-hilarious manner is the day this is likely to get fixed

**Comment by Zth--** [comment]
I once discovered a bug with vines and unwrenched hydroponics trays too. I think I forgot to report.

**Comment by Kurfursten** [comment]
It was already done but not in the way you'd expect.

I've seen this used to spread kudzu to multiple places very quickly. Keep seeds in your plant bag then drag an unwrenched tray behidn you.

**Comment by Zth--** [comment]
I once discovered a bug with vines and unwrenched hydroponics trays too. I think I forgot to report.

**Comment by Kurfursten** [comment]
I once discovered a doublepost by Zth-- that he forgot to delete

**Comment by Zth--** [comment]
I once discovered a bug with vines and unwrenched hydroponics trays too. I think I forgot to report it.

You need vines in a tray. Then if you unwrench it and start pulling while running, you'll get vines everywhere

**Comment by despotate** [comment]
One way to exploit this would be to unwrench all the trays in the vox outpost (all 26 of them) and plant chickenshrooms, then stuff them all into a crate. Instant denitrogenation of any room where you open the crate.

Also you posted the same thing three times, delete the older posts.

**Comment by Zth--** [comment]
I was doing _ctrl+enter_ instead of _shift+enter_. ¯\\_(ツ)_/¯

**Comment by Exxion** [comment]
What about
Crate full of one thousand plasma-emitting plants
(Well 30, because that's the limit a crate can hold, but still)

I wonder if you could replace a gas miner

**Comment by Zth--** [comment]
I think the plants would kill themselves after a while if they're not high pressure resistant. By the way how much pressure do modified plants resist?

**Comment by despotate** [comment]
Yeah you can do wild things with enough exotic seeds, fuck with the pressure, gas composition, or temperature. Doesn't help that crates essentially place the trays into stasis. Kind of a symptom of how really limitless seeds are, clip-planting is instant and doesn't take that much time to do. A possible tweak to this would be to have plants start at 0 health instead of `endurance` so you can't just instantly clip them for replanting.

As zth brings up, plants SHOULD kill themselves over time, but they heal ~4 per 10 seconds if fed and watered and only take ~2 damage per damage type (pressure, temperature, gas) so even if you made them fuck with both temperature and pressure they would only remain at the same health until they start dying of old age. Which again brings up balance problems with hydroponics - plants rarely ever die except to gross incompetence and even when they DO die you'll have extracted at least one harvest out of them. I mean, let's say you fuck up majorly and plant a plant into a tray that lacks pressure, heat, two necessary gases, and light - that's only 6 damage per age, meaning it'll get to 17 age (which can be between 1-3 harvests) before it finally dies. What exactly is even growing the plant at that point?

The most annoying symptom of plants' ridiculous endurance and lifespan values is how hard it is to kill them when you want to. It takes ten or so hatchet strikes to down those chanterelles that just popped out as a result of a weed invasion, and plant-b-gon will kill any plant you choose at the expense of also killing the next two that you plant in that tray. Easiest way I've found to quickly kill a plant is to use plant clippers on them, they do 40 damage per clip with no attack delay and usually bring the plant down to two or three strikes to down it.

**Comment by Exxion** [comment]
I'm pretty sure you can fix the toxicity by dumping a shitload of water into the tray, but I'm not entirely sure if that works for Plant-B-Gon
It's easier to just not let weeds grow in the first place

**Comment by despotate** [comment]
I used a bad example. The only real times I want to kill a plant is when all my trays are occupied and it's the most useless one, or more often, the tomato just mutated into a blood tomato instead of a blue tomato.

Plant-b-gon can be mixed with around 10 or 15 units of water but that's too much hassle to deal with and too much time to wait. And as much as I'd like to always be there for my trays and unweed them before bad goes to worse, Traders can't change farmbot controls with their IDs so you can't really do anything to stop the weedsplosion before you return from your trading expedition.

**Comment by Exxion** [comment]
Farmbots don't even work 80% of the time anyway

Yeah, there are some weird plant properties for sure

**Comment by despotate** [comment]
What do you mean farmbots don't work? They seem to water and deweed my plants just fine though sometimes they have pathing issues and get stuck. Maybe you're not closing the lids on your nutriment bottles before feeding them to farmbots? They can't remove pests on their own either.

**Comment by Exxion** [comment]
#5494 
#12111 

They're broken as shit and if you can get them to work consistently then you're like the luckiest person in existence

Last time I tried to use one was quite a while ago, but I don't think they've been touched since then

**Comment by Zth--** [comment]
I'm sure farmbots _at least_ water the plant.

The underlying problem here is the lack of documentation of the equations and the mechanics of some parts of the code, in this case hydroponics. We should bring numbers and equations (thanks @despotate ) in discussions like these so we can really balance instead of guessing how some things work.

**Comment by Kurfursten** [comment]
@despotate If you put water in the tray it will get rid of the toxin
